### PR TITLE
Update translation: hu-HU

### PR DESCRIPTION
**Pull request by rdbende**

### CODE REVIEW

File: Files/MultilingualResources/Files.hu-HU.xlf [hunk 1]
```diff
@@ -2352,31 +2352,31 @@
         </trans-unit>
         <trans-unit id="SidebarFavorites" translate="yes" xml:space="preserve">
           <source>Favorites</source>
-          <target state="new">Favorites</target>
+          <target state="new">Kedvencek</target>
```
Comment (tsvietOK) [inline Files/MultilingualResources/Files.hu-HU.xlf:2355]: All translated strings should be marked as `translated`.

Comment (rdbende) [reply Files/MultilingualResources/Files.hu-HU.xlf]: Oh, I was a little careless. Thank you for the warning!

File: Files/MultilingualResources/Files.hu-HU.xlf [hunk 2]
```diff
@@ -1576,11 +1576,11 @@
         </trans-unit>
         <trans-unit id="PropertyPromotionUrl" translate="yes" xml:space="preserve">
           <source>Promotion Url</source>
-          <target state="new">Promotion Url</target>
+          <target state="translated">Promotion Url</target>
```
Comment (tsvietOK) [inline Files/MultilingualResources/Files.hu-HU.xlf:1579]: Only translated strings should be marked as translated. Keep untranslated strings with state 'new'.

**Review by tsvietOK — CHANGES_REQUESTED** [review]

File: Files/MultilingualResources/Files.hu-HU.xlf (hunk 2 above)
Comment (rdbende) [reply Files/MultilingualResources/Files.hu-HU.xlf]: Ok, I'm starting to understand.

File: Files/Strings/hu-HU/Resources.resw [hunk 3]
```diff
@@ -1418,6 +1418,9 @@
   </data>
   <data name="BaseLayoutContextFlyoutSortByOriginalPath.Text" xml:space="preserve">
     <value>Eredeti elérési út</value>
+  </data>
+    <data name="PreemptiveCacheParallelLimit.Header" xml:space="preserve">
```
Comment (tsvietOK) [inline Files/Strings/hu-HU/Resources.resw:1422]: Looks like you edited this file manually. This file is auto generated from `Files.hu-HU.xlf`, so no need to edit it manually. To generate updated file just run project build or just exclude this file from PR and we will do it by ourselves.

Comment (tsvietOK) [reply Files/Strings/hu-HU/Resources.resw:1422]: For more information about translation, visit our [documentation](https://files-community.github.io/docs/#/articles/translating-files).

Comment (rdbende) [reply Files/Strings/hu-HU/Resources.resw]: Ok, I restored the file, I don't want to cause any more stupidity, I'd rather leave the update of Files/Strings/hu-HU/Resources.resw to you.

**Review by tsvietOK — APPROVED** [review]

**Comment by nvi9** [comment]
@rdbende do you have any reference of why "licensz" is the correct translation instead of "licenc"?

https://helyesiras.mta.hu/helyesiras/default/suggest?q=licensz

**Pull request merged**